components/patient-data/ui/src/main/resources/PhenoTips/GeneClass.xml

```diff
@@ -114,7 +174,7 @@
       <unmodifiable>0</unmodifiable>
       <validationMessage/>
       <validationRegExp/>
-      <values>candidate=Candidate|rejected_candidate=Rejected candidate|solved=Confirmed causal|carrier=Carrier|rejected=Negative</values>
+      <values>candidate=Candidate|candidate>novel_disease=Novel disease gene|candidate>novel_phen=Known disease gene with novel phenotype|umc=Uncertain molecular cause|umc>vus=Known disease gene with a variant of unknown significance|umc>msv=Known disease gene missing a second variant|rejected_candidate=Rejected candidate|solved=Confirmed causal|carrier=Carrier|rejected=Tested negative</values>
```

Comment:
We decided on shorter labels for some of the new status entries. The Jira issue has been updated accordingly.

```suggestion
      <values>candidate=Candidate|candidate>novel_disease=Novel disease gene|candidate>novel_phen=Known disease gene with novel phenotype|umc=Uncertain molecular cause|umc>vus=Known disease gene with a VUS|umc>msh=Known disease gene missing a second hit|rejected_candidate=Rejected candidate|solved=Confirmed causal|carrier=Carrier|rejected=Tested negative</values>
```